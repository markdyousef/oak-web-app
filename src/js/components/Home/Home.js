import React, { Component } from 'react';

import logo from 'img/clai-logo.png';

import css from './Home.css';


class Home extends Component {
    static propTypes = {}

    constructor() {
        super();
        this.state = {};
    }

    render() {
        return (
            <div className={css.container}>
                <h1>Home</h1>
                <img src={logo} role="presentation" />
            </div>
        );
    }
}

export default Home;
