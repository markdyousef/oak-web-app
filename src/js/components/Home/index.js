const React = require('react');

/* class HomePage extends React.Component {
    render() {
        return (
            <div>Home</div>
        );
    }
} */


function HomePage() {
    return (
        <div>Home</div>
    );
}

export default HomePage;
