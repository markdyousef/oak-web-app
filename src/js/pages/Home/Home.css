:root {
    --primary-color: #5ed892;
    --main-black-color: #131517;
    --light-black-color: #2d3438;
    --dark-gray-color: #737a80;
    --light-gray-color: #f2f2f2;
}

.container {
    width: 100%;
    display: flex;
    flex-direction: column;
}

.stats {
    width: 100%;
    height: 200px;
    background-color: #fff;
    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
}
