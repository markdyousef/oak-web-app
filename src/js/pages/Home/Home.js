// @flow
import React, { Component, PropTypes } from 'react';
import Collections from '../../components/Collections';
import TopNav from '../../components/TopNav';

import css from './Home.css';

class Home extends Component {
    static propTypes = {};
    static defaultProps = {};
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <div className={css.container}>
                <TopNav />
                <div className={css.categories}>
                    <Collections />
                </div>
            </div>
        );
    }
}

export default Home;
