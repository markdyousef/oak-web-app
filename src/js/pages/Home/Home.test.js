import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

it('renders correctly', () => {
    const shallowComponent = shallow(<Home />)
    expect(shallowComponent).toMatchSnapshot();
});
