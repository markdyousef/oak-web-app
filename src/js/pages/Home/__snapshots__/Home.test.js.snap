exports[`test renders correctly 1`] = `
<div
  className={undefined}>
  <nav
    className={undefined}>
    <div
      className={undefined} />
    <div
      className={undefined}>
      <a
        className={undefined}
        onClick={[Function]}
        style={Object {}}>
        <div
          className={undefined}>
          <img
            alt="profile"
            src="//style.anu.edu.au/_anu/4/images/placeholders/person.png" />
        </div>
      </a>
    </div>
  </nav>
  <div
    className={undefined}>
    <div
      className={undefined}>
      <div
        className={undefined}>
        <div
          className={undefined}>
          <div
            className={undefined}>
            KNOWLEDGESTATS
          </div>
        </div>
        <div
          className={undefined}>
          <button
            className=" "
            onClick={[Function]}>
            <div>
              <label
                className={undefined}
                htmlFor="text">
                Add Collection
              </label>
            </div>
          </button>
        </div>
      </div>
      <div
        className={undefined}>
        <a
          onClick={[Function]}
          style={Object {}}>
          <div
            className={undefined}>
            <div
              className={undefined}>
              <h1>
                Design
              </h1>
            </div>
          </div>
        </a>
        <a
          onClick={[Function]}
          style={Object {}}>
          <div
            className={undefined}>
            <div
              className={undefined}>
              <h1>
                Frontend
              </h1>
            </div>
          </div>
        </a>
        <a
          onClick={[Function]}
          style={Object {}}>
          <div
            className={undefined}>
            <div
              className={undefined}>
              <h1>
                Strategy
              </h1>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
